notebooks/237-typo-detector/README.md

```diff
@@ -0,0 +1,58 @@
+# Convert and Optimize Typo Detector with OpenVINO™
+
+Typo detection in AI is a process of identifying and correcting typographical errors in text data using machine learning algorithms. The goal of typo detection is to improve the accuracy, readability, and usability of text by identifying and correcting mistakes made during the writing process.
+
+A typo detector takes a sentence as an input and identify all typographical errors such as misspellings and homophone errors.
+
+This tutorial provides how to use the [Typo Detector](https://huggingface.co/m3hrdadfi/typo-detector-distilbert-en) from the [Hugging Face Transformers](https://huggingface.co/docs/transformers/index) library to perform the above task.
+
+The model has been pretrained on the [NeuSpell](https://github.com/neuspell/neuspell) dataset.
+
+<img src=https://user-images.githubusercontent.com/80534358/224564463-ee686386-f846-4b2b-91af-7163586014b7.png>
+
+</br>
+
+The notebook provides two methods to use the typo detector in the OpenVino runtime. I've demonstrated both methods so that you can experience both the loading of compiled models and loading models in other frameworks and converting them.
```

Comment:
```suggestion
The notebook provides two methods to run the inference of typo detector with OpenVINO runtime, so that you can experience both calling the API of Optimum with OpenVINO Runtime included, and loading models in other frameworks, converting them to OpenVINO IR format, and running inference with OpenVINO Runtime.
```